pytorch/training/docker/1.11/py3/cu113/Dockerfile.gpu

```diff
@@ -178,12 +167,12 @@ RUN ompi_info --parsable --all | grep mpi_built_with_cuda_support:value \
     typing \
     h5py \
     requests \
```

Comment:
why is future removed?

Reply:
It's has a vulnerability. But future already comes with python as a module, code isn't affected.